src/Nancy.Authentication.Stateless/StatelessAuthentication.cs

```diff
@@ -74,16 +74,10 @@ public static void Enable(INancyModule module, StatelessAuthenticationConfigurat
 
             return context =>
             {
-                try
-                {
-                    context.CurrentUser = configuration.GetUserIdentity(context);
-                    return context.Response;
-                }
-                catch (Exception)
-                {
-                    return context.Response;
-                }
+                context.CurrentUser = configuration.GetUserIdentity(context);
+
+                return null;

```

Comment:
Cool. I was unaware. Maybe we can improve the framework in the future where we done have to give special meaning to "null". It worries me any time I see it used.